library/Zend/Form/Element.php

```diff
@@ -199,6 +199,8 @@ class Zend_Form_Element implements Zend_Validate_Interface
      */
     protected $_validators = [];
 
+    protected array $_validatorBreakChainOnFailures = [];
```

Comment:
FWIW, the backwards-compatible fix here is to leave out the `array` keyword, since that feature was only added in PHP 7.4